client/db/src/bench.rs

```diff
@@ -109,6 +151,86 @@ impl<B: BlockT> BenchmarkingState<B> {
 		));
 		Ok(())
 	}
+
+	fn add_whitelist_to_tracker(&self) {
```

Comment:
correct me if I am wrong: by whitelisting, basically we already assume that they have been read once? there's no inherent meaning to being whitelisted other than that.

Reply:
Yes, this key is free to read and write to, and does not count in DB tracking.

Things like BlockNumber, the Sender Account, Events, etc...

Reply:
Okay, one question then: won't this cause confusion with the read-write count? Say I submit a tx that has no reads and writes. Won't the read/write count of my tx then be equal to all the whitelisted ones?

Reply:
(Either way I think it is all okay, I am just trying to make it clear for myself.)

Reply:
no, I dont increment the read/write count when I do whitelisting. So assuming nothing is actually read/written from, you will get 0 reads and 0 writes.